pkg/lang/python/aws_runtime/dispatcher_fargate.py.tmpl

```diff
@@ -59,7 +60,14 @@ def start_proxy_server():
         function = getattr(module_obj, function_name, None)
         if not function:
             raise Exception(f"couldn't find function: {module_name}.{function_name}")
-        result = function(*params)
+        param_args = ()
+        param_kwargs = {}
+        args_spec = inspect.getfullargspec(function)
+        if args_spec.varkw:
+            param_kwargs, params = params[-1], params[:-1]
+        if args_spec.varargs:
+            param_args, params = params[-1], params[:-1]
+        result = function(*params, *param_args, **param_kwargs)
```

Comment:
Won't this break if you have both `*args` and `**kwargs` in the same spec? Both assignments are using `params[-1], params[:-1]` on the same list.

Reply:
(same goes for the other dispatcher)